_maps/templates/boarding/syndicate/mako_carrier.dmm

```diff
@@ -28,8 +28,14 @@
 /area/ruin/powered/nsv13/boarding_interior)
 "aV" = (
 /obj/structure/displaycase,
+/obj/item/documents/syndicate,
 /turf/open/floor/carpet/red,
 /area/ruin/powered/nsv13/boarding_interior)
+"aW" = (
+/obj/structure/closet/crate,
+/obj/effect/spawner/lootdrop/ruinloot/medical,
```

Comment:
```suggestion
/obj/effect/spawner/lootdrop/maintenance/six,
```
the medical lootdrop has some pretty gamer equipment, it's probably better to just use maintenance ones for these (and maybe add a single medical ruin loot spawner in the medbay?)
Images for reference:
![image](https://user-images.githubusercontent.com/43698041/208088447-a6c89e1d-8425-45bb-9526-d2d3a73a1622.png)
![image](https://user-images.githubusercontent.com/43698041/208088598-6cad6207-ce88-4cae-94db-07c867f07241.png)